x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/comments-template.json

```diff
@@ -0,0 +1,46 @@
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "comments": {
+          "type": "nested",
+          "properties": {
+            "id": {
+              "type": "keyword"
+            },
+            "author": {
+              "properties": {
+                "email": {
+                  "type": "keyword"
+                },
+                "name": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "id": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "content": {
+              "type": "text"
+            },
+            "replies_to": {
+              "type": "keyword"
+            },
```

Comment:
Isn't that superseded by `author`? I feel like the only kind of comment/content where a "replyTo" dissociated from the user id/principal makes sense are emails (or email threads) so not sure it makes sense to have it in a generic comment template?

Reply:
Maybe I need a different field name. This is intended to represent which `comments.id` _this_ comment is in direct reply to. So that you could reverse-engineer the thread tree, and not just have to display all comments in timestamp order.